solr/CHANGES.txt

```diff
@@ -91,6 +91,8 @@ Improvements
   /api/collections/collName/snapshots/snapshotName`, listed at `GET /api/collections/collName/snapshots`, and deleted at
   `DELETE /api/collections/collName/snapshots/snapshotName`. (John Durham via Jason Gerlowski)
 
+* SOLR-16752: Reduce attack surface and size for Docker image (Quang-Cuong Bui) 
```

Comment:
Since the size improvement is neglible, reword somewhat?
```suggestion
* SOLR-16752: Docker: Leave out optional apt packages, slightly reducing image size and lowering attach surface (Quang-Cuong Bui) 
```

Reply:
Thanks. Updated the recommended title.